### DISCUSSION TITLE
Feature proposal: LXQtile

### DISCUSSION THREAD

**Discussion by githubisnonfree** [lxqt/lxqt#2137] [Ideas]
I've studied tiling window managers, and one thing strikes me: the paradigm is excellent, but tiling window managers are inaccessible to normies.

Why not add a tiling mode to LXQt? Make it behave like Qtile.

see: qtile.org

I understand that the codebases of LXQt and Qtile are very different, so I'm not proposing to merge these codebases, but I think this would be great:

A hotkey, or button (placed on the panel), can be used to switch between normal LXQt, and... LXQtile! LXQtile mode would act exactly like Qtile and offer similar configurability. However, it would be hybrid.

It would combine the ease of use (for most normal users) with the power of a tiling window manager. I wonder, would people like such a feature?

**Comment by tsujan** [comment]
Tiling is done by WM. LXQt doesn't have a WM — it's WM agnostic. See https://lxqt-project.org/about/

**Comment by githubisnonfree** [comment]
yeah i was under the impression that most lxqt setups use openbox

i suppose then that this is a matter of configuration, as opposed to the lxqt project managing it. i envision: a hybrid window manager where you can switch between openbox behaviour and qtile behaviour, interchangeably with a key press

hybrid window manager. it's probably been done before, but i thought of it today anyway

**Reply by tsujan** [reply]
The distro you use may have chosen Openbox but any DE-agnostic WM can be used with LXQt.

**Reply by githubisnonfree** [reply]
Close this issue then. Disregard my proposal.

**Reply by tsujan** [reply]
You correctly opened a page under Discussions. Discussions can't be closed and we don't remove them.
